Source/SaveExtension/Private/Serialization/SlotDataTask_Loader.cpp

```diff
@@ -369,8 +369,12 @@ void USlotDataTask_Loader::RespawnActors(const TArray<FActorRecord>& Records, co
 	// Respawn all procedural actors
 	for (const auto& Record : Records)
 	{
-		SpawnInfo.Name = Record.Name;
-		World->SpawnActor(Record.Class, &Record.Transform, SpawnInfo);
+		auto pExistingActor = StaticFindObjectFast(nullptr, SpawnInfo.OverrideLevel, Record.Name, true);
```

Comment:
This function should only receive actor records that don't exist in the level, and that's checked at "PrepareLevel". Should this check be there? Maybe the problem is with that function.

Reply:
Moved to PrepareLevel.

I encounter this issue because at beginning I never pass save filter without BakeAllowedClasses 
 in OnMapLoaded for BP classes. The code in this PR should fix this problem, but I think we still need this, because the patch after release the game that may change SaveFilter, so we must ensure existing actor in level should not in ActorsToSpawn.

Reply:
Makes sense. Good idea.